src/ControlSystem/Tags/MeasurementTimescales.hpp

```diff
@@ -24,7 +26,31 @@ template <class Metavariables, typename ControlSystem>
 struct ControlComponent;
 /// \endcond
 
-namespace control_system::Tags {
+namespace control_system {
+/*!
+ * \ingroup ControlSystemGroup
+ * \brief Calculate the measurement timescale based on the damping timescale,
+ * update fraction, and DerivOrder of the control system
+ *
+ * The update timescale is \f$\tau_\mathrm{update} = \alpha_\mathrm{update}
+ * \tau_\mathrm{damp}\f$ where \f$\tau_\mathrm{damp}\f$ is the damping timescale
+ * (from the TimescaleTuner) and \f$\alpha_\mathrm{update}\f$ is the update
+ * fraction (from the controller). For an Nth order control system, the averager
+ * requires at least N measurements in order to perfom its finitite differencing
+ * to calculate the derivatives of the control error. This implies that the
+ * largest the measurement timescale can be is \f$\tau_\mathrm{m} =
+ * \tau_\mathrm{update} / N\f$. To ensure that we have sufficient measurements,
+ * we calculate the measurement timescales as \f$\tau_\mathrm{m} =
+ * \tau_\mathrm{update} / (N+1)\f$
+ */
+template <size_t DerivOrder>
+DataVector calculate_measurement_timescales(
```

Comment:
You can usually omit prefixes like `calculate_`, `compute_` etc from function names, since they don't really add any information. Consider removing in a future PR.